README.md

```diff
@@ -6,11 +6,27 @@
 npm install
 
 npm run watch
+
+node server/index.js
 ```
+
 Then open the project in Visual Studio Code.
 Go to `extension.ts` then press F5.
 
-## Commands
+## To install as an extension
+
+```
+npm install -g @vscode/vsce
+
+cd live-interviewing
+
+vsce package
+```
+
```

Comment:
In addition to the release notes discussed on Slack, can you add this:

> When releasing a new version, be sure to also replace the existing `latest` release and tag with a new version. When attaching the `.vsix` file to the `latest` release, ensure the file is named `live-session-vscode-extension.vsix` without a version number.

And for the versioned releases, we can just upload the file as generated with the version number to keep it simple (no need to rename it to `hatchways-v0.0.x`)